packages/gatsby/src/utils/webpack.config.js

```diff
@@ -345,7 +345,10 @@ module.exports = async (
         configRules = configRules.concat([
           {
             oneOf: [
-              rules.cssModules(),
+              rules.cssModules({
+                // enable smaller css classnames
+                localIdentName: `[hash:base64:5]`,
```

Comment:
I've seen collisions in the past with short class names. Is there an easy escape hatch for this setting? Also, some people (myself included) prefer to have readable class names even with CSS modules to make debugging easier.

Reply:
it will only be used in prod, gatsby should do the fast thing here and not care about debugability in prod.